pkgs/development/python-modules/standard-mailcap/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  lib,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "standard_mailcap";
+  version = "3.13.0";
+  disabled = pythonOlder "3.13";
+  src = fetchPypi {
```

Comment:
`fetchFromGitHub` is preferred

Reply:
But the github repo has many subfolders with different "dead batteries" in each subfolder. So I would have to pull one subfolder from the github repo. Is that still preferred?

Reply:
Yes.

Reply:
I tried this and it looks much more complicated to me.
With fetchPypi I have
~~~nix
src = fetchPypi {
  pname = "standard_mailcap";
  inherit version;
  hash = "sha256-Ge15VdvqzLNei7BbK1RDzlXB+TKoy+elwT1C+dtPSZo=";
};
~~~
whereas with fetchFromGitHub I have
~~~nix
src = fetchFromGitHub {
  owner = "youknowone";
  repo = "python-deadlib";
  tag = "v3.13.0";
  hash = "sha256-9mtQi5ufxP6xRonTrFC3oWFpWLbJraAmdQYozP3evgc=";
  sparseCheckout = [ "mailcap" ];
};
preBuild = "cd mailcap";
~~~

Just to be absolutely sure: fetchFromGtHub is still preferred in this situation?